pkgs/development/python-modules/qdldl/default.nix

```diff
@@ -1,25 +1,56 @@
 {
   lib,
   buildPythonPackage,
-  fetchPypi,
+  fetchFromGitHub,
   cmake,
   numpy,
   pybind11,
   setuptools,
   scipy,
   pytestCheckHook,
+  qdldl,
 }:
 
 buildPythonPackage rec {
   pname = "qdldl";
   version = "0.1.7.post5";
   pyproject = true;
 
-  src = fetchPypi {
-    inherit pname version;
-    hash = "sha256-CxOZ4cSbW+1arI/WPvCKtwjTQMN/tCb+ABKLwfNrKG4=";
+  src = fetchFromGitHub {
+    owner = "osqp";
+    repo = "qdldl-python";
+    tag = "v${version}";
+    hash = "sha256-XHdvYWORHDYy/EIqmlmFQZwv+vK3I+rPIrvcEW1JyIw=";
   };
 
+  # use up-to-date qdldl for CMake v4
+  postPatch = ''
```

Comment:
This would probably be nicer as an actual `.patch`.

Reply:
I'm not sure how to pass `${lib.getDev}/include` in that case. Any recommendation ?

Reply:
You can use `replaceVars`.